cmd/minikube/cmd/docker-env.go

```diff
@@ -324,9 +323,10 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 			// docker-env on containerd depends on nerdctld (https://github.com/afbjorklund/nerdctld) as "docker" daeomn
 			// and nerdctld daemon must be used with ssh connection (it is set in kicbase image's Dockerfile)
 			// so directly set --ssh-host --ssh-add to true, even user didn't specify them
-			sshAdd = true
 			sshHost = true
+		}
 
+		if sshHost {
```

Comment:
Instead of having two `if sshHost` blocks, could we move this down to the second block?

Reply:
Yes.
Now it has been done